lib/hca/enrollment_eligibility/service.rb

```diff
@@ -57,7 +57,8 @@ def get_ezr_data(icn)
       end
 
       # rubocop:disable Metrics/MethodLength
-      def lookup_user(icn)
+      # The 'year' param is compared to the financial 'incomeYear' on file for the Veteran
+      def lookup_user(icn, year = nil)
```

Comment:
year is always supposed to be the current year?

Reply:
Yes. However, to make the code testable, I injected the `year` dependency. Otherwise, a truthy test this year will become falsy next year.

Reply:
you could use timecop.freeze in the tests instead of adding a parameter that isn't necessary

Reply:
Oh okay, I didn't know about that. I'll add that in now. Thanks!